app/controllers/returns/ReturnsController.scala

```diff
@@ -63,7 +64,11 @@ class ReturnsController @Inject()(val sessionService: SessionService,
         AccountingPeriodForm.form.bindFromRequest.fold(
           errors => Future.successful(BadRequest(accountingPeriodPage(errors))),
           success => returnsService.saveStaggerStart(success) flatMap { _ =>
-            Future.successful(Redirect(controllers.flatratescheme.routes.JoinFlatRateSchemeController.show))
```

Comment:
I thought this page was meant to be shown to every flow, this only puts it into a very specific one that goes through account periods.